src/convert.rs

```diff
@@ -28,6 +28,7 @@ pub(crate) trait ToOrderedInteger<T> {
     fn map_to(&self) -> T;
 }
 
+#[allow(dead_code)]
 trait FromOrderedInteger<T> {
     fn map_from(input: T) -> Self;
 }
```

Comment:
What is an "ordered integer"?

Reply:
We'll have to check with @coderdan as to what this does. It's currently not exported anywhere, so that's why clippy was complaining about it.